pkg/lib/bundle/generate.go

```diff
@@ -379,7 +343,7 @@ func GenerateDockerfile(mediaType, manifests, metadata, copyManifestDir, copyMet
 	fileContent += fmt.Sprintf("LABEL %s=%s\n", MetadataLabel, metadata)
 	fileContent += fmt.Sprintf("LABEL %s=%s\n", PackageLabel, packageName)
 	fileContent += fmt.Sprintf("LABEL %s=%s\n", ChannelsLabel, channels)
-	fileContent += fmt.Sprintf("LABEL %s=%s\n\n", ChannelDefaultLabel, chanDefault)
```

Comment:
If `channelDefault` is an empty string, this label will be empty. Is that ok/desirable?

Reply:
I think it is OK to not have a defaultChannel value AKA empty string because in annotation generation function, there is a case where inferred defaultChannel is set to empty string. @Bowenislandsong Can you double-confirm this?

Reply:
Wouldn't it be better to just not set this label at all?

Reply:
yes. Later when added to Index, the default behavior will kick in (right now the default behavior is to add the first channel as default).

Reply:
In that case, wouldn't it be better/more obvious to the user to write a label containing the first channel as default if a default isn't explicitly set?